null_safety_examples/null_safety_codelab/bin/promotion_exceptions.dart

```diff
@@ -0,0 +1,11 @@
+int getLength(String? str) {
+  // Try throwing exception here if `str` is null.
+  if (str == null) {
+  	return Exception("String is null");
+  }
```

Comment:
i couldn't find a way to get this to pass. i hit this error, WDYT?

```
A value of type 'Exception' can't be returned from function 'getLength' because it has a return type of 'int'
```

Reply:
looks like i just need to learn about throwing exceptions. :-) i'll fix this momentarily.

Reply:
you want to `throw`, not `return`. Also, you might need to use a specific exception (like `UnimplementedException`), but maybe not.

Reply:
If it makes you feel any better, I did this a _lot_ at one point.

Reply:
:D